docs/tutorials/text_prediction/beginner.md

```diff
@@ -140,6 +140,18 @@ for val, color in [(0, 'red'), (1, 'blue')]:
 plt.legend(loc='best')
 ```
 
+### Continuous Training
+:label:`sec_textprediction_continuous_training`
+
+You can also load a predictor and call `.fit()` again for continuous training.
```

Comment:
user might not know what "for continuous training" means. May be clearer to say:
"to continue training the same predictor with additional data"

Reply:
Revised.